_config.yml

```diff
@@ -136,7 +136,7 @@ offline:
         origin: maxcdn.bootstrapcdn.com
 
 # Deployment
-## Docs: http://zespia.tw/hexo/docs/deployment.html
+## Docs: https://hexo.io/docs/one-command-deployment
```

Comment:
There is no `/docs/deployment` equivalent, so I just picked the last deployment page because it is more generic